tests/acceptance/features/apiSpaces/moveSpaces.feature

```diff
@@ -0,0 +1,243 @@
+    Then the HTTP status code should be "502"
+    Examples:
+      | permissions |
+      | 31          |
+      | 17          |
+
+
+  Scenario Outline: User move a file from space shares jail with different role (permissions) to space personal
+    Given user "Brian" has created folder "/testshare"
+    And user "Brian" has uploaded file with content "testshare content" to "/testshare/testshare.txt"
+    And user "Brian" has shared folder "/testshare" with user "Alice" with permissions "<permissions>"
+    And user "Alice" has accepted share "/testshare" offered by user "Brian"
+    When user "Alice" moves file "/testshare/testshare.txt" from space "Shares Jail" to "testshare.txt" inside space "Personal" using the WebDAV API
+    Then the HTTP status code should be "502"
+    Examples:
+      | permissions |
+      | 31          |
+      | 17          |
+
+
+  Scenario Outline: User move a file from space shares jail with role editor to space project with different role
+    Given the administrator has given "Brian" the role "Space Admin" using the settings api
+    And user "Brian" has created a space "Project" with the default quota using the GraphApi
+    And user "Brian" has shared a space "Project" to user "Alice" with role "<role>"
+    And user "Brian" has created folder "/testshare"
+    And user "Brian" has uploaded file with content "testshare content" to "/testshare/testshare.txt"
+    And user "Brian" has shared folder "/testshare" with user "Alice" with permissions "31"
+    And user "Alice" has accepted share "/testshare" offered by user "Brian"
+    When user "Alice" moves file "/testshare/testshare.txt" from space "Shares Jail" to "testshare.txt" inside space "Project" using the WebDAV API
+    Then the HTTP status code should be "502"
+    Examples:
+      | role    |
+      | manager |
+      | editor  |
+      | viewer  |
+
+
+  Scenario Outline: User move a file from space shares jail with viewer role to space project with different role
+    Given the administrator has given "Brian" the role "Space Admin" using the settings api
+    And user "Brian" has created a space "Project" with the default quota using the GraphApi
+    And user "Brian" has shared a space "Project" to user "Alice" with role "<role>"
+    And user "Brian" has created folder "/testshare"
+    And user "Brian" has uploaded file with content "testshare content" to "/testshare/testshare.txt"
+    And user "Brian" has shared folder "/testshare" with user "Alice" with permissions "17"
+    And user "Alice" has accepted share "/testshare" offered by user "Brian"
+    When user "Alice" moves file "/testshare/testshare.txt" from space "Shares Jail" to "testshare.txt" inside space "Project" using the WebDAV API
+    Then the HTTP status code should be "502"
+    Examples:
+      | role    |
+      | manager |
+      | editor  |
+      | viewer  |
```

Comment:
Maybe these two can be merged as well? 
```suggestion
  Scenario Outline: User moves a file from space shares jail with role editor and viewer to space project with a different role
    Given the administrator has given "Brian" the role "Space Admin" using the settings api
    And user "Brian" has created a space "Project" with the default quota using the GraphApi
    And user "Brian" has shared a space "Project" to user "Alice" with role "<role>"
    And user "Brian" has created folder "/testshare"
    And user "Brian" has uploaded file with content "testshare content" to "/testshare/testshare.txt"
    And user "Brian" has shared folder "/testshare" with user "Alice" with permissions "<permission>"
    And user "Alice" has accepted share "/testshare" offered by user "Brian"
    When user "Alice" moves file "/testshare/testshare.txt" from space "Shares Jail" to "testshare.txt" inside space "Project" using the WebDAV API
    Then the HTTP status code should be "502"
    Examples:
      | role    | permission |
      | manager | 31 |
      | editor  | 31 |
      | viewer  | 31 |
      | manager | 17 |
      | editor  | 17 |
      | viewer  | 17 |

```
Similar with very other scenarios that can be merged